src/topology/instances/ennreal.lean

```diff
@@ -595,6 +595,16 @@ protected lemma tsum_apply {ι α : Type*} {f : ι → α → ennreal} {x : α}
   (∑' i, f i) x = ∑' i, f i x :=
 tsum_apply $ pi.summable.mpr $ λ _, ennreal.summable
 
+lemma tsum_sub {f:ℕ → ennreal} {g:ℕ → ennreal}
+(h₁ : (∑' i, g i) < ∞) (h₂ : g ≤ f) : (∑' i, (f i - g i)) = (∑' i, f i) - (∑' i, g i) :=
```

Comment:
```suggestion
lemma tsum_sub {f : ℕ → ennreal} {g : ℕ → ennreal} (h₁ : (∑' i, g i) < ∞) (h₂ : g ≤ f) : 
  (∑' i, (f i - g i)) = (∑' i, f i) - (∑' i, g i) :=
```
Please do indent lemma statements if they span multiple lines. I also like to have the conclusion on a line by itself.